src/main/java/com/puppycrawl/tools/checkstyle/checks/javadoc/SummaryJavadocCheck.java

```diff
@@ -402,6 +433,22 @@ private static boolean isSummaryTag(DetailNode javadoc) {
                 && SUMMARY_TEXT.equals(child[1].getText());
     }
 
+    /**
+     * Checks if the first tag inside ast is summary tag.
+     *
+     * @param javadoc root node.
+     * @return {@code true} if first tag is summary tag.
+     */
+    private static boolean isInlineReturnTag(DetailNode javadoc) {
```

Comment:
isSummaryTag is almost the same method.

please make `isInlineTag(javadoc, String tagName)` to combine them to be the same method.

Reply:
done